### ISSUE TITLE
Documentation Versioning

### ISSUE THREAD

**Issue by jeremylt** [CEED/libCEED#765]
The version number in the Julia documentation is based on the release tag in the main libCEED repo. We should figure out how to use the actual Julia package version. Also, we should make sure the Rust documentation and future Python documentation have intended versioning behaviour.

**Comment by jeremylt** [comment]
It looks like libCEED.jl is now tracking the libCEED version number. If this will continue to be the case, then we can close this issue.

**Comment by pazner** [comment]
The version of `libCEED_jll` (the Julia package that provides the prebuilt libCEED binaries) should always track the version number of libCEED.

We *could* do the same with LibCEED.jl (the Julia interface package), but I thought we decided against that. In any case, its current version is 0.1.1, not 0.9.0.

**Comment by jedbrown** [comment]
I think the issue is that if you go to the version tab at https://ceed.exascaleproject.org/libCEED-julia-docs/dev/, you see v0.9, which is the `libCEED_jll` version number, but not the `LibCEED.jl` version. But the website claims that this is `LibCEED.jl` documentation. Is this version identifier being obtained from Git tags, and if so, is there a different way to specify it?

**Comment by jeremylt** [comment]
Yeah, the version number comes from git tags and I haven't found a way around this. Ours is probably an uncommon use case.

**Comment by jedbrown** [comment]
Apache Arrow had this issue for a bit, then the Julia implementation split out of the shared repository. I'm fine closing this issue. If we have lots of Julia users at some point (or developers wanting to make frequent Julia releases), we can revisit and/or split `LibCEED.jl` into its own repository. Please reopen if/when needed.
